src/fastoad/models/aerodynamics/aerodynamics_high_speed.py

```diff
@@ -17,19 +17,17 @@
 import openmdao.api as om
```

Comment:
There are some " FAST - Copyright (c) 2016 ONERA ISAE " at the begining of several files in the aerodynamic module.